StudioCore/Editor/ProjectSettings.cs

```diff
@@ -49,7 +50,21 @@ public static ProjectSettings Deserialize(string path)
         {
             var jsonString = File.ReadAllBytes(path);
             var readOnlySpan = new ReadOnlySpan<byte>(jsonString);
-            return JsonSerializer.Deserialize<ProjectSettings>(readOnlySpan);
+            ProjectSettings settings;
+            try
+            {
+                settings = JsonSerializer.Deserialize<ProjectSettings>(readOnlySpan);
+                if (settings == null)
+                    throw new Exception("JsonConvert returned null");
+            }
+            catch (Exception e)
+            {
+                MessageBox.Show($"{e.Message}\n\nProject.json cannot be loaded, and will be deleted. Please create a new project.",
+                    $"Project Load Error", MessageBoxButtons.OK);
+                File.Delete(path);
```

Comment:
I don't think automatically deleting a file without giving the user an option to confirm it at all is a good idea...

Reply:
Fair. The reason why was because config and keybind handled it in a functionally similar way (though by overwriting on save vs deleting the file, I suppose) but those probably should've prompted in the first place too.